README.md

```diff
@@ -227,11 +241,22 @@ Hooks can be defined on a group level (`foo`, `bar`) so that they apply to all c
 * A fatal error will be raised at startup when 2 group-inherited hooks conflict. This is not the case in the previous example; even though `foo` and `bar` both contain `service2`, the hooks they declare are disjoint.
 
 The following hooks are currently available:
+* `pre-build`: Executed before building an image
+* `post-build`: Executed after building an image
 * `pre-start`: Executed before starting or running a container
 * `post-start`: Executed after starting or running a container
 * `pre-stop`: Executed before stopping, killing or removing a running container
 * `post-stop`: Executed after stopping, killing or removing a running container
 
+Every hook will have the name of the container for which this hook runs available as the environment variable `CRANE_HOOKED_CONTAINER`.
+
+### Container Prefixes
+It is possible to prefix containers with a global `--prefix` flag, which is just
```

Comment:
Why not? :)

I personally prefer to wrap the lines since that makes the source file more readable and doesn't affect the display.